projects/demo/src/modules/decorators/required-setter/import/example-decorator.txt

```diff
@@ -4,7 +4,7 @@
     'Should be integer number more than min value',
 )
 set quantity(quantity: number) {
-    this.items = new Array(quantity).fill(Math.floor(Math.random() * Math.floor(100)));
+    this.items = new Array(quantity ?? 0).fill(Math.floor(Math.random() * Math.floor(100)));
```

Comment:
This is unnecessary. Type is `number` and nullish coalescing will not do anything with `NaN` so there's no point.

Reply:
Aww, thx, forgot to change that.